types/react-signature-canvas/react-signature-canvas-tests.tsx

```diff
@@ -0,0 +1,46 @@
+import ReactSignatureCanvas from 'react-signature-canvas';
+import SignaturePad from 'signature_pad';
+import * as React from 'react';
+
+const ReactSignatureCanvasNoOptions: JSX.Element = <ReactSignatureCanvas />;
+
+const ReactSignatureCanvasDotSizeFunctionOptions: JSX.Element = <ReactSignatureCanvas dotSize={() => 4} />;
+
+const ReactSignatureCanvasAllOptions: JSX.Element = (
+    <ReactSignatureCanvas
+        dotSize={2}
+        minWidth={2}
+        maxWidth={2}
+        throttle={2}
+        minDistance={2}
+        backgroundColor="blue"
+        penColor="red"
+        velocityFilterWeight={2}
+        onBegin={(event: MouseEvent) => {}}
+        onEnd={(event: MouseEvent) => {}}
+        canvasProps={{ title: 'canvas' }}
```

Comment:
Some frequently used `canvasProps` are `width`, `height`, and `style` or `className` (CSS `backgroundColor` is a topic that pops up occasionally) -- would probably be good to use those instead